DOCS-737 - Update enrichment info (#5143)
SumoLogic/sumologic-documentation: docs/security/threat-intelligence/threat-indicators-in-cloud-siem.md

For more information, see [hasThreatMatch](/docs/cse/rules/cse-rules-syntax/#hasthreatmatch).

## View threat indicator labels in the Cloud SIEM UI

> Entities are automatically enriched with indicator data from the [sources you ingest](/docs/security/threat-intelligence/about-threat-intelligence/#ingest-threat-intelligence-indicators) (but not from the **_sumo_global_feed_cs** global feed). When a match to a threat indicator in your sources is found, labels showing the entity "reputation" will be displayed throughout the Cloud SIEM UI:

Reviewer: ```suggestion
Entities are automatically enriched with indicator data from the [sources you ingest](/docs/security/threat-intelligence/about-threat-intelligence/#ingest-threat-intelligence-indicators), excluding the **_sumo_global_feed_cs** global feed. When a match to a threat indicator in your sources is found, labels showing the entity "reputation" will be displayed throughout the Cloud SIEM UI:
```

| Reputation value | Label | Icon |
|:--|:--|:--|
| **Malicious** | <img src={useBaseUrl('img/cse/indicator-malicious-label.png')} alt="Malicious label" width="100"/> | <img src={useBaseUrl('img/cse/indicator-malicious-icon.png')} alt="Malicious icon" width="30"/> |
| **Suspicious** | <img src={useBaseUrl('img/cse/indicator-suspicious-label.png')} alt="Suspicious label" width="110"/> | <img src={useBaseUrl('img/cse/indicator-suspicious-icon.png')} alt="Suspicious icon" width="30"/> |
| **Not Flagged** | <img src={useBaseUrl('img/cse/indicator-notflagged-label.png')} alt="Suspicious label" width="125"/> | None |

> **Not Flagged** is not the default value (which is no indicator at all). Cloud SIEM will not automatically determine the indicator value; enrichments must explicitly set it.

Reviewer: ```suggestion
**Not Flagged** is not the default value (which is no indicator at all). Cloud SIEM does not automatically assign an indicator value—enrichments must explicitly define it.
```

:::note

> Although Cloud SIEM automatically enriches *signals* with information from your sources in the Sumo Logic threat intelligence datastore, it does not enrich *records* like it did with [custom intelligence sources](/docs/cse/administration/create-custom-threat-intel-source/). Even if there is a match per the rule expression, the indicator body will not be attached to the record. Instead, the signal should have a list of matched indicator IDs and the indicators will also be visible on the matched entities. 

Reviewer: ```suggestion
Cloud SIEM automatically enriches *signals* with information from your sources in the Sumo Logic threat intelligence datastore. However, unlike [custom intelligence sources](/docs/cse/administration/create-custom-threat-intel-source/), it does not enrich *records*. Even if a match occurs based on the rule expression, the indicator body is not attached to the record. Instead, the signal should contain a list of matched indicator IDs, and the indicators are visible on the matched entities.
```
